packages/dds/tree/src/test/simple-tree/api/getSimpleSchema.spec.ts

```diff
@@ -9,10 +9,26 @@ import {
 	getSimpleSchema,
 	NodeKind,
 	SchemaFactory,
+	type SimpleLeafNodeSchema,
+	type SimpleNodeSchema,
+	type SimpleObjectFieldSchema,
+	type SimpleObjectNodeSchema,
 	type SimpleTreeSchema,
 } from "../../../simple-tree/index.js";
 import { ValueSchema } from "../../../core/index.js";
 
+const simpleString: SimpleLeafNodeSchema = {
```

Comment:
Nit: Given that these are used as the expected output (derived from input defined within each test), it seems a bit odd to me to have them shared here and not have the input schema shared. The correspond to static definitions at the end of the day, so this isn't a huge issue, but it might be nice to just directly use the static leaf schema definitions?

This is a nitpick. Feel free to disregard.

Reply:
The tests do actually share what these are derived from in some cases, for example the first two tests using it uses schemaFactory.string, a shared constant.

Reply:
I think most or all of them use that, I just didn't check every one.